code/workspaces/web-app/src/components/stacks/StackCardActions.js

```diff
@@ -75,6 +75,14 @@ export const PureStackCardActions = ({ stack, openStack, deleteStack, editStack,
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
+        {stack.type === 'rshiny' && <PermissionWrapper userPermissions={userPermissions} permission={deletePermission}>
```

Comment:
Just checking this is only meant to be applied to `rshiny` apps

Reply:
Yes at the moment, don't want to expose anything else as of yet.